integration-tests/itest/cli/cli.go

```diff
@@ -176,16 +190,51 @@ func GetStreamletPod(status *AppStatus, streamlet string) *StreamletPod {
 	return nil
 }
 
+// GetPods retrieves the pods corresponding to the given streamlet from an AppStatus instance
+func GetPods(status *AppStatus, streamlet string) []string {
+	var res []string
+	for _, entry := range status.StreamletPods {
+		if entry.Streamlet == streamlet {
+			res = append(res, entry.Pod)
+		}
+	}
+	return res
+}
+
 // GetOneOfThePodsForStreamlet returns a pod associated to the streamlet runtime
 func GetOneOfThePodsForStreamlet(app App, streamlet string) (pod string, err error) {
 	status, err := Status(app)
 	if err != nil {
 		return
 	}
-	streamletPod := GetStreamletPod(&status, streamlet)
-	if streamletPod == nil {
+	pods := GetPods(&status, streamlet)
+	if pods == nil {
 		err = fmt.Errorf("could not find entry for streamlet [%s]", streamlet)
 		return
 	}
-	return streamletPod.Pod, nil
+	return pods[0], nil
+}
+
+// GetPodsForStreamlet retrieves the pods that correspond to the given streamlet.
+func GetPodsForStreamlet(app App, streamlet string) (pods []string, err error) {
+	status, err := Status(app)
+	if err != nil {
+		return nil, err
+	}
+	return GetPods(&status, streamlet), nil
+}
+
+// PollUntilExpectedPodsForStreamlet retrieves the pods that correspond to the given streamlet when
+// the pod count is or becomes equal to the expected count.
```

Comment:
```suggestion
// PollUntilExpectedPodsForStreamlet polls the pods that correspond to the given streamlet until 
// the pod count becomes equal to the expected count.
```

Reply:
Thanks for the suggestion. I find that "retrieves" better describes the functionality of the method because it gives a return value.

Reply:
The more important part is that this method *keeps polling* until it gets to the expected count.
I found this sentence a little strange:
> retrieves the pods that correspond to the given streamlet when the pod count is or becomes equal to the expected count.
Note the `is or becomes equal to ` part.  

I think the comment would be clearer as:

> PollUntilExpectedPodsForStreamlet continues to retrieve pods that correspond to the given streamlet until the pod count is equal to the expected count.

Or something more to that effect.

Reply:
I see where the confusion is: 
I'm using "retrieves" to convey "the value it returns" while you are reading it as the action it's doing internally. 
I see the ambiguity in this. I replaced 'retrieves' with 'returns' and added a note about polling. 
I hope it's clear now.